helsingborg-stad/app-mitt-helsingborg
# Feat/remove unused code

## Explain the changes you’ve made
Removed files that weren't used

## Explain why these changes are made
Code that is not used is not worth keeping

## Explain your solution
Removed code that weren't imported by any other file

## How to test


1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Run the application as usual, no crashes or other errors should occur 

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
